bhyve-api/src/structs.rs

```diff
@@ -277,3 +277,33 @@ pub struct vm_run_state {
     pub sipi_vector: u8,
     pub _pad: [u8; 3],
 }
+
+pub const VM_MAX_NAMELEN: usize = 128;
+pub const VM_MAX_SEG_NAMELEN: usize = 128;
+
+// Allocate guest memory segments from existing reservoir capacity, rather than
+// attempting to create transient allocations.
+pub const VCF_RESERVOIR_MEM: u64 = 1;
```

Comment:
IMO it would be useful to label this as a flag, somehow. That's not immediately apparently without reading the corresponding illumos change.

Reply:
Added additional comment context